src/webui/webapplication.cpp

```diff
@@ -656,7 +656,9 @@ void WebApplication::sessionStart()
     QNetworkCookie cookie(C_SID, m_currentSession->id().toUtf8());
     cookie.setHttpOnly(true);
     cookie.setPath(QLatin1String("/"));
-    header(Http::HEADER_SET_COOKIE, cookie.toRawForm());
+    QByteArray cookieRawForm = cookie.toRawForm();
+    cookieRawForm.append("; SameSite=Strict");
```

Comment:
I think this should be: `if (m_isCSRFProtectionEnabled) "strict" else none`.

Reply:
This does seem to fall under that option well. I've added the check.